scripts/sql_connect

```diff
@@ -12,7 +12,7 @@ set -o pipefail
 # DEBUG = 2
 declare -i desired_log_level=2
 declare -a valid_environment_targets=( dev prod )
-/
+
```

Comment:
not sure what the slash does/did in an otherwise empty line, is this an intentional change?

Reply:
yeah it was intentional. I suspect it was a typo from a previous pr